bin/phoenix_utils.py

```diff
@@ -77,7 +77,7 @@ def findClasspath(command_name):
     return tryDecode(subprocess.Popen(command, shell=True, stdout=subprocess.PIPE).stdout.read())
 
 def setPath():
-    PHOENIX_CLIENT_JAR_PATTERN = "phoenix-client.jar"
+    PHOENIX_CLIENT_JAR_PATTERN = "phoenix-client*[!s][!t].jar"
```

Comment:
This will still match sources:

ls -l phoenix-client*[!s][!t].jar
-rw-r--r--  1 stoty  staff   75651719 Nov 17 11:04 phoenix-client-5.1.0-SNAPSHOT-sources.jar
-rw-r--r--  1 stoty  staff  144017266 Nov 17 11:04 phoenix-client-5.1.0-SNAPSHOT.jar

This would not:

[stoty@IstvanToth-MBP15:~/workspaces/apache-phoenix/phoenix/phoenix-client-parent/phoenix-client/target (master)$]ls -l phoenix-client*[!s].jar
-rw-r--r--  1 stoty  staff  144017266 Nov 17 11:04 phoenix-client-5.1.0-SNAPSHOT.jar

Reply:
I have modified it，please have a look